src/ros_filter.cpp

```diff
@@ -1135,7 +1138,7 @@ void RosFilter::loadParams()
 
         topic_subs_.push_back(node_->create_subscription<
             geometry_msgs::msg::PoseWithCovarianceStamped>(
-            pose_topic, pose_callback));
+            pose_topic, 10, pose_callback));
```

Comment:
As well here, you have declares for other related parameters `_relative`, you need one for `_queue_size`